<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoWidget</class>
 <widget class="QDialog" name="videoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="CVVideoWidget" name="videoLayer" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="previousButton">
       <property name="text">
        <string>previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gotoButton">
       <property name="text">
        <string>goto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="frameInput"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="setE1Button">
       <property name="text">
        <string>set E1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gotoE1Button">
       <property name="text">
        <string>goto E1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setE2Button">
       <property name="text">
        <string>set E2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gotoE2Button">
       <property name="text">
        <string>goto E2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentFrameLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="event1Label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="event2Label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QListWidget" name="e1List">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>300</width>
         <height>200</height>
        </size>
       </property>
       <property name="flow">
        <enum>QListView::LeftToRight</enum>
       </property>
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="e2List">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>300</width>
         <height>200</height>
        </size>
       </property>
       <property name="flow">
        <enum>QListView::LeftToRight</enum>
       </property>
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="text">
        <string>accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="declineButton">
       <property name="text">
        <string>decline</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CVVideoWidget</class>
   <extends>QWidget</extends>
   <header>qtpromotion</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>videoWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>declineButton</sender>
   <signal>clicked()</signal>
   <receiver>videoWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
